Pod/Classes/SEGAmplitudeIntegration.m

```diff
@@ -21,6 +21,10 @@ - (id)initWithSettings:(NSDictionary *)settings andAmplitude:(Amplitude *)amplit
             self.traitsToIncrement = [NSSet setWithArray:self.settings[@"traitsToIncrement"]];
         }
 
+        if (self.settings[@"traitsToSetOnce"] != (id)[NSNull null]) {
```

Comment:
Do we need the cast here? Also does this handle `nil`?

Reply:
I thought we determined that settings.traitsToSetOnce could be @[] or null, not nil

Reply:
But does this check handle all those cases? It seems to me it would only handle the null case.